extension/src/dvcYaml/index.ts

```diff
@@ -0,0 +1,130 @@
+import { join } from 'path'
```

Comment:
It would be good to rename all `dvcYaml` directories to `pipelines` as that is the high-level concept that these YAML files belong to. We will be able to add DAG-related code, etc to these directories later.

Reply:
I would try to avoid adding files to `extension/src/fileSystem` as well if possible.

Reply:
Thanks! I was looking for the right name but I thought it would be better to just be literal and ask about it later.